### ISSUE TITLE
Uncaught TypeError: Cannot call a class as a function

### ISSUE THREAD

**Issue by andyyou** [gajus/react-css-modules#43]
I want to check a question that if I use the React version before `React 0.14 rc` (I use `0.13.3`), I have to downgrade to 3.3.0?

**Comment by likidu** [comment]
Got the same error here..

I'm using decorator, it used to work like a charm but now it's somehow broken.

**Comment by enqvistab** [comment]
+1

**Comment by gajus** [comment]
Does this only happen when using decorators?

If not, please provide a code example (the class declaration that fails).

If yes, a temporary workaround is to use non-decorator syntax. If confirmed, I will provide a fix within the next two hours.

**Comment by likidu** [comment]
I tried both w/out decorator and none is working. Here is my code sample:

With decorator:

```javascript
// App.js
import React, { Component } from 'react';
import cssModule from 'react-css-modules';
import styles from './styles.css';
import Test from './Test';

@cssModule(styles)
export default class App extends Component {
  render() {
    return (
      <div>
        <h1>Hello, world.</h1>
        <Test />
      </div>
    );
  }
}
```

It's sub component uses non-decorator syntax:

```javascript
import React, { Component } from 'react';
import cssModule from 'react-css-modules';
import styles from './styles.css';

class Test extends Component {
  render() {
    return (
      <div>
        <p>Hello, React</p>
      </div>
    );
  }
}

export default cssModule(Test, styles);
```

The `styles.css`:

```css
body {
  padding-top: 5rem;
  color: red;
}

h1 {
  color: blue;
}
```

**Comment by likidu** [comment]
Tested on latest release: 3.5.1

**Comment by bdefore** [comment]
I'm also having this problem, with a decorator, on 3.5.1

**Comment by gajus** [comment]
It appears that build was not included in the latest version (https://github.com/gajus/react-css-modules/pull/44). Please try 3.5.2.

**Comment by andyyou** [comment]
Thanks a lot, It fixed :+1:

**Comment by bdefore** [comment]
Also fixed for me on 3.5.2. Thanks!

**Comment by likidu** [comment]
Fixed now. Thanks for such quick fix:)
